Cargo.lock

```diff
@@ -2296,7 +2288,7 @@ dependencies = [
  "iroh-quinn-proto",
  "iroh-quinn-udp",
  "iroh-relay",
- "n0-future",
+ "n0-future 0.2.0",
```

Comment:
Another new duplicate dependency.
Looking at the rest of this file, seems like it's used in n0-snafu and net-tools.